modules/core/include/opencv2/core/hal/intrin_neon.hpp

```diff
@@ -1642,7 +1642,7 @@ inline int v_signmask(const v_uint64x2& a)
 #if CV_NEON_AARCH64
     const int64x2_t signPosition = {0,1};
     uint64x2_t v0 = vshlq_u64(vshrq_n_u64(a.val, 63), signPosition);
-    uint64_t t0 = vaddvq_u64(v0);
+    int t0 = (int)vaddvq_u64(v0);
```

Comment:
I was able to cross compile for ARM64 Windows and can reproduce only this warning (MS VS 2022). I propose to revert other changes and merge the PR for now.